src/toil_scripts/lib/files.py

```diff
@@ -78,3 +125,41 @@ def consolidate_tarballs_job(job, fname_to_id):
                         tarinfo.name = os.path.join(output_name, fname, os.path.basename(tarinfo.name))
                         f_out.addfile(tarinfo, fileobj=f_in_file)
     return job.fileStore.writeGlobalFile(out_tar)
+
+
+def untargz(input_targz_file, untar_to_dir):
```

Comment:
This function would work for plain .tar archives as well, correct? If so, let's document that.